setup.py

```diff
@@ -36,27 +36,26 @@ def finalize_options(self):
 setup(
     author="Jonas Teuwen, George Yiasemis",
     author_email="j.teuwen@nki.nl, g.yiasemis@nki.nl",
-    python_requires=">=3.8",
+    python_requires=">=3.9",
     classifiers=[
         "Development Status :: 5 - Production/Stable",
         "Intended Audience :: Developers",
         "OSI Approved :: Apache Software License",
         "Natural Language :: English",
         "Programming Language :: Python :: 3",
-        "Programming Language :: Python :: 3.8",
-        "Programming Language :: Python :: 3.9",
         "Programming Language :: Python :: 3.10",
+        "Programming Language :: Python :: 3.11",
     ],
     description="DIRECT - Deep Image REConsTruction - is a deep learning" " framework for MRI reconstruction.",
     entry_points={
         "console_scripts": [
             "direct=direct.cli:main",
         ],
     },
-    setup_requires=["numpy", "cython"],
+    setup_requires=["numpy>=1.21.2", "cython>=3.0"],
     install_requires=[
         "numpy>=1.21.2",
-        "h5py==3.3.0",
+        "h5py==3.11.0",
         "omegaconf==2.1.1",
         "torch>=1.10.2",
         "torchvision",
```

Comment:
Still old version of torch, can you update? Also add explicit versions for torchvision. Same with OmegaConf

Reply:
Done